pages/about.md

```diff
@@ -0,0 +1,40 @@
+<script>{
+	"title": "About QUnit",
+	"pageTemplate": "page-contentfull.php"
+}</script>
+
+QUnit is a powerful, easy-to-use JavaScript unit testing framework. It's used by the jQuery, jQuery UI, jQuery Mobile and many other projects and is capable of testing any generic JavaScript code, including itself!
+
+## Vision and Goals
+
+### Simple setup
+
+Setting up the first unit test with QUnit should be super simple, requiring as little setup as possible.
+
+### Immediate, detailed feedback
+
+When a test or assertion fails, QUnit should provide feedback to the developer as fast as possible, with enough detail to quickly figure out the underlying issue.
+
+### (What else?)
+
+???
+
+## QUnit Team
+
+There's a lot of work that goes into making QUnit. Between API design, implementation, ticket triage, bug fixing, developer relations, infrastructure, and everything else, most of the work is done by volunteers. We'd like to recognize the most prominent contributors below, for a full list of all contributors, see the [authors list](https://github.com/jquery/qunit/blob/master/AUTHORS.txt).
+
+### [Leo Balter](https://twitter.com/leobalter) — Project Lead
+
+(todo)
```

Comment:
New version: 

> Leo is a software engineer at Bocoup. Born in Rio de Janeiro, Brazil, he's now based in Boston, MA, USA. Leo actively contributes to QUnit since 2013. He also wrote some of [the official JavaScript tests](http://github.com/tc39/test262/) for ES2015, also known as ES6.

*GH comments are not made for version control*